src/pages/index.tsx

```diff
@@ -24,16 +43,15 @@ export default function Home() {
 
   class Decks extends React.Component<DeckProps, DeckState> {
     render() {
-      const listItems = this.props.decks.map((deck) => (
-        <Link href={`/review/${deck.id}`}>
+      return this.props.decks.map((deck) => (
+        <Link href={`/review/${deck.id}`} key={deck.id}>
```

Comment:
console was complaining when there was no key

Reply:
Nice! Did you notice this when you ran `npm run dev`? I don't pay enough attention to the warnings and such, but running `npm run lint` gave me that as an error, too. Thanks!